lib/encoder_unittest.py

```diff
@@ -214,6 +221,20 @@ def test_OtherFormatter(self):
       formatter=encoder.OptionFormatter(prefix='-', infix=' '))
     self.assertEqual('-foo foo -xyz abc', valueset.ToString())
 
+  def test_RandomlyRemoveParameterSuccessfully(self):
+    config = encoder.OptionValueSet(
+      encoder.OptionSet(encoder.Option('foo', ['foo', 'bar'])),
```

Comment:
Yes, this is laziness. Unfortunately my current emacs seems to do the Wrong Thing when indenting after a parenthesis, and I haven't found a setting to change it.